### DISCUSSION TITLE
secrets: inherit is not working in GH Enterprise

### DISCUSSION THREAD

**Discussion by maheshsmartcow** [community/community#29406] [Enterprise] [answered]
May I check why `secrets: inherit` option is not available in GitHub enterprise ? I think its a pain to declare all the secrets in the reusable workflow and again in the caller, then what is the DRY about ?

**Comment by juliandunn** [comment, marked as answer]
It is available there. You need to be on [GitHub Enterprise Server 3.6](https://docs.github.com/en/enterprise-server@3.6/admin/release-notes) or higher to use `secrets: inherit`.
